android/app/src/main/AndroidManifest.xml

```diff
@@ -1,36 +1,33 @@
-<manifest xmlns:android="http://schemas.android.com/apk/res/android"
-          package="com.spruceid.app.credible" android:installLocation="auto">
-
-    <uses-permission android:name="android.permission.INTERNET"/>
-    <uses-permission android:name="android.permission.CAMERA"/>
```

Comment:
Probably good to restore the previously used permissions, otherwise release builds might not work properly or be uploadable to the store.